Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/DataGridSamplePage.xaml.cs

```diff
@@ -0,0 +1,283 @@
+			{
+				e.RowGroupHeader.PropertyValue = (item?.IsWatered ?? false) ? "Yes" : "No";
+			}
+			else
+			{
+				e.RowGroupHeader.PropertyValue = (item?.FruitOrVegetable == FruitOrVegetableEnum.Fruit) ? "Fruit" : "Vegetable";
+			}
+		}
+
+		private void DataGrid_PreparingCellForEdit(object sender, DataGridPreparingCellForEditEventArgs e)
+		{
+			if (e.EditingElement is CalendarDatePicker calendar)
+			{
+				calendar.IsCalendarOpen = true;
+			}
+			else
+			if (e.EditingElement is ComboBox comboBox)
+			{
+				comboBox.IsDropDownOpen = true;
+			}
+		}
+	}
+
+	public class DataGridSamplePageViewModel : ViewModelBase
+	{
+		public int ColumnHeaderHeight { get => GetProperty<int>(); set => SetProperty(value); }
+		public int MaxColumnWidth { get => GetProperty<int>(); set => SetProperty(value); }
+		public int MinColumnWidth { get => GetProperty<int>(); set => SetProperty(value); }
+		public bool AreRowDetailsFrozen { get => GetProperty<bool>(); set => SetProperty(value); }
+		public bool AreRowGroupHeadersFrozen { get => GetProperty<bool>(); set => SetProperty(value); }
+		public bool CanUserSortColumns { get => GetProperty<bool>(); set => SetProperty(value); }
+		public bool CanUserReorderColumns { get => GetProperty<bool>(); set => SetProperty(value); }
+		public bool CanUserResizeColumns { get => GetProperty<bool>(); set => SetProperty(value); }
+		public bool IsReadOnly { get => GetProperty<bool>(); set => SetProperty(value); }
+		public SolidColorBrush AlternatingRowForeground { get => GetProperty<SolidColorBrush>(); set => SetProperty(value); }
+		public SolidColorBrush AlternatingRowBackground { get => GetProperty<SolidColorBrush>(); set => SetProperty(value); }
+		public DataGridGridLinesVisibility GridLinesVisibility { get => GetProperty<DataGridGridLinesVisibility>(); set => SetProperty(value); }
+		public DataGridHeadersVisibility HeadersVisibility { get => GetProperty<DataGridHeadersVisibility>(); set => SetProperty(value); }
+		public DataGridSelectionMode SelectionMode { get => GetProperty<DataGridSelectionMode>(); set => SetProperty(value); }
+		public List<SolidColorBrush> AvailableColors { get => GetProperty<List<SolidColorBrush>>(); set => SetProperty(value); }
+		public List<DataGridGridLinesVisibility> GridLinesVisibilityOptions { get => GetProperty<List<DataGridGridLinesVisibility>>(); set => SetProperty(value); }
+		public List<DataGridHeadersVisibility> HeadersVisibilityOptions { get => GetProperty<List<DataGridHeadersVisibility>>(); set => SetProperty(value); }
+		public List<DataGridSelectionMode> SelectionModeOptions { get => GetProperty<List<DataGridSelectionMode>>(); set => SetProperty(value); }
+
+		public string RowGroupHeader { get => GetProperty<string>(); set => SetProperty(value); }
+
+		public IEnumerable DataGridItems { get => GetProperty<IEnumerable>(); set => SetProperty(value); }
+		public FruitOrVegetableEnum[] FruitOrVegetableEnumValues;
+
+		// This is currently causes bug on WASM, removed until fixed
```

Comment:
```suggestion
		// This currently causes a bug on WASM, removed until fixed
```
Please add in the comment the related link for this specific issue

Reply:
Right, thanks! I forgot to add it, I have added it to the code. To link it with this pull request, it's this one https://github.com/unoplatform/uno/issues/11184